match.py

```diff
@@ -0,0 +1,177 @@
+"""Switch match."""
+
+import ipaddress
+
+from pyof.v0x01.common.flow_match import FlowWildCards
+
+from kytos.core import log
+
+from .settings import IPV4
+
+
+def format_request(request):
+    """Format user request to match function format."""
+    args = {}
+    for key, value in request.items():
+        args[key] = value
+    return args
+
+
+def do_match(flow_to_install, version, stored_flow_dict):
+    """Match a packet against this flow."""
+    if version == 0x01:
+        return match10(stored_flow_dict, flow_to_install)
+    elif version == 0x04:
+        return match13_no_strict(flow_to_install, stored_flow_dict)
+    return None
```

Comment:
I think this should raise an error if the OpenFlow version is not supported.

Reply:
Resolved in https://github.com/kytos/flow_manager/pull/117/commits/47c31e83eb8603d1625077c354265c6b849838be